src/bika/lims/browser/publish/emailview.py

```diff
@@ -511,14 +511,14 @@ def render_email_template(self, template, template_context=None):
         """
 
         # allow to add translation for initial template
-        template = self.context.translate(_(template))
+        template = self.context.translate(_(safe_unicode(template)))
         recipients = self.email_recipients_and_responsibles
         if template_context is None:
             template_context = {
-                "recipients": "<br/>".join(recipients),
+                "recipients": "<br/>".join(map(safe_unicode, recipients)),
```

Comment:
Please provide the joining element `"<br/>"` also as unicode `u"<br/>"`

Reply:
my bad.. corrected